test/custom-elements/index.ts

```diff
@@ -16,9 +16,9 @@ const page = `
 
 const assert = fs.readFileSync(`${__dirname}/assert.js`, 'utf-8');
 
-describe('custom-elements', function() {
+describe('custom-elements', function () {
```

Comment:
```suggestion
describe('custom-elements', function() {
```

Reply:
Do we want to revert these spacing changes? I know it's out of scope of the PR, but I believe it's auto-formatted from prettier, which would be nice to be resolved.

Reply:
> it's auto-formatted from prettier

Yes.

I haven't looked at it closely yet, but such a suggestion could be prevented with the Lint configuration.
If possible, can I create a PR to update the Lint configuration?

Refer: 
https://eslint.org/docs/rules/space-after-function-name

Reply:
Anyway we can have a conversation about it outside of this PR. So just I will revert these formatting changes.

Reply:
This repo doesn't use prettier, but I guess you're suggesting that it was installed in the IDE and that's where it's coming from? I'd be fine with aligning our eslint config to match prettier. I think the rule would be:
```
    "space-before-function-paren": ["error", {
        "anonymous": "always"
    }],
```

Reply:
Oh I thought this repo had prettier. Might've though of something else. Reverting looks good then.

Reply:
As a side note, this formatting was performed by the TypeScript Language Server on VSCode.

> I think the rule would be:

Thank you. When I have time, I will check the impact of this rule adding and if it looks OK, I will send a PR.